alt_e2eshark/e2e_testing/backends.py

```diff
@@ -160,12 +138,6 @@ def __init__(self, *, device="local-task", hal_target_device="llvm-cpu", extra_a
             self.extra_args = [
                 "--iree-hip-target=gfx90a",
             ]
```

Comment:
FYI, we've been pulling this from an environment variable in other tests. See https://github.com/nod-ai/SHARK-Platform/pull/373#discussion_r1822904806 for some discussion

Reply:
Yeah, that's a good idea. It's a bit verbose to need to write out all the flags repeatedly.